src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1940,6 +1942,18 @@ because xbuild doesn't support framework reference assemblies.
       <Output TaskParameter="ResolvedSymbols" ItemName="ResolvedSymbols" />
       <Output TaskParameter="ResolvedDoNotPackageAttributes" ItemName="_ResolvedDoNotPackageAttributes" />
   </ResolveAssemblies>
+  <Hash ItemsToHash="@(ResolvedAssemblies)">
+	<Output TaskParameter="HashResult" PropertyName="_ResolvedUserAssembliesHash" />
+  </Hash>
```

Comment:
I went and double-checked, item metadata is not included in the hash: https://github.com/Microsoft/msbuild/blob/0263a588f3db16abe4ae4a051213f4809e8b55c2/src/Tasks/Hash.cs#L13

This looks good 👍